src/backend.rs

```diff
@@ -1,18 +1,23 @@
 use std::path::{Path, PathBuf};
 use std::collections::HashMap;
-use eyre::{Result, anyhow};
+use anyhow::{Result, anyhow};
```

Comment:
Why use anyhow instead?

Reply:
The switch from eyre to anyhow is to ensure compatibility with the ? operator.
anyhow's error type implements std::error::Error and can be used in contexts where ? operator is applied